postgresql.conf.jinja

```diff
@@ -316,6 +316,40 @@ log_min_duration_statement = '1500ms'
 log_line_prefix = '%m [%p] %q%u@%d '
 log_timezone = 'Europe/Berlin'
 
+# Controls which SQL statements are logged. Valid values are none (off), ddl, mod, and all (all statements).
+# ddl logs all data definition statements, such as CREATE, ALTER, and DROP statements. mod logs all ddl
+# statements, plus data-modifying statements such as INSERT, UPDATE, DELETE, TRUNCATE, and COPY FROM. PREPARE, 
+# EXECUTE, and EXPLAIN ANALYZE statements are also logged if their contained command is of an appropriate type. 
+{%- if log_statement is defined %}
+log_statement = '{{ log_statement }}'
+{%- endif %}
+
+# This parameter enables the logging collector, which is a background process that captures log
+# messages sent to stderr and redirects them into log files. This approach is often more useful than logging to syslog,
+# since some types of messages might not appear in syslog output. (One common example is dynamic-linker failure messages;
+# another is error messages produced by scripts such as archive_command.) This parameter can only be set at server start.
+#
+# NOTE: It is possible to log to stderr without using the logging collector; the log messages will just go to
+# wherever the server's stderr is directed. However, that method is only suitable for low log volumes, since
+# it provides no convenient way to rotate log files. Also, on some platforms not using the logging collector can
+# result in lost or garbled log output, because multiple processes writing concurrently to the same log file can
+# overwrite each other's output.
+# NOTE: The logging collector is designed to never lose messages. This means that in case of extremely high load,
+# server processes could be blocked while trying to send additional log messages when the collector has fallen behind.
+# In contrast, syslog prefers to drop messages if it cannot write them, which means it may fail to log some messages
+# in such cases but it will not block the rest of the system.
+{%- if logging_collector is defined %}
+logging_collector = {{ logging_collector }}
```

Comment:
`on` will be interpreted as `True` - therefore this will result in a broken `postgresql.conf`. 

I'm currently using `true` and `false` here in the configuration and then handle the setting in the configuration file like so:


```jinja
{%- if wal_log_hints %}
wal_log_hints = on
{%- else %}
wal_log_hints = off
{%- endif %}
```

Reply:
For consistency, I'd then also adapt the `pillar.example` to say `logging_collector: true` - that's how the other binary settings are handled as well.